### PR TITLE
Cloudfront auth cognito

### PR DESCRIPTION
**Pull request by mslipets**
Updated and augmented according to incoming PR's (suggestions)

### CODE REVIEW

**Comment by mslipets** [comment]
Hey @lazzurs, @jmcgeheeiv,
first of all thanks for advice!
Please take a look and comment this PR (if interested),
this PR intended to improve this module and adhere to your suggestions.

File: example/main.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ module "cloudfront_auth" {
 
   bucket_name                    = "private.example.com"
   region                         = "eu-west-1"
-  cloudfront_acm_certificate_arn = "${aws_acm_certificate.cert.arn}"
+  cloudfront_acm_certificate_arn = aws_acm_certificate.cert.arn
 }
```
Comment (jmcgeheeiv) [inline example/main.tf:15]: Did you add the additional required arguments? I do not see them in the diff.

**Review by jmcgeheeiv — COMMENTED** [review]
Please also see my revised proposed documentation for `authz` in my `variables.tf`.

File: example/main.tf (hunk 1 above)
Comment (mslipets) [reply example/main.tf:15]: Thanks for heads up, added with latest commits.

File: variables.tf [hunk 2]
```diff
@@ -111,5 +120,6 @@ variable "geo_restriction_whitelisted_locations" {
 
 variable "bucket_access_roles_arn_list" {
   type        = list(string)
-  description = "A Role ARN which granted RW rights to bucket (to be used by instance profiles)"
+  default     = []
+  description = "A Role ARN which granted RW rights to bucket (to be used by instance profiles in order to upload/cleanup bucket content)"
```
Comment (jmcgeheeiv) [inline variables.tf:124]: "grants"

**Pull request merged**